VLogProcessor/libs/mac_client/opt/vertica/include/verticaodbc.h

```diff
@@ -1,32 +0,0 @@
-/*
```

Comment:
@Sorrow17 Do you know what were the files in `VLogProcessor/libs` for? Doesn't deleting all of them make any trouble for us in the future?